firebase-installations/src/main/java/com/google/firebase/installations/time/SystemClock.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.time;
+
+/** @hide */
+public class SystemClock implements Clock {
+  @Override
+  public long now() {
+    return System.currentTimeMillis();
```

Comment:
Please link to some Java documentation like this one:
https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()